go/test/endtoend/vtgate/queries/derived/derived_test.go

```diff
@@ -56,6 +56,7 @@ func TestDerivedTableWithOrderByLimit(t *testing.T) {
 }
 
 func TestDerivedAggregationOnRHS(t *testing.T) {
+	utils.SkipIfBinaryIsBelowVersion(t, 18, "vtgate")
```

Comment:
For this test, it makes sense. Support for the query was added in v18 via https://github.com/vitessio/vitess/pull/13750 after we saw it was failing on https://github.com/vitessio/vitess/issues/11703.